### ISSUE TITLE
Seach Functions dont work? [MySQL]

### ISSUE THREAD

**Issue by ablac** [SimonOrJ/CoreProtect-Lookup-Web-Interface#26]
When using the web interface, If i select just an Action, it shows up with the information requested, but if I select a user it says "An Unknown Error Occurred" It does this for Coordinates, Radius, Worlds, Users, Materials, entities,  Date/Time, and even Limit... So basically the entire website is useless unless i want just a massive flood of data in the map.

**Comment by thunderlight411** [comment]
same behavior as in https://www.spigotmc.org/threads/coreprotect-lookup-web-interface.173574/#post-3811046

**Comment by ablac** [comment]
Yes same Behavior, Willing to allow testing with my site as well https://angel-gaming.com/bans

Secondly I'm running MariaDB 10.3.17-MariaDB MariaDB Server
Core Protect Versions 2.18.2

**Comment by chuushi** [comment]
I'll start looking into it next week.  I believe it has to do with the old versions of MySQL/MariaDB server not understanding the query.

Until then, if you can, try updating to the latest MySQL server version.

**Comment by ablac** [comment]
@chuushi Thats very interesting, So i downloaded the new MariaDB Repo, and Yum Updated to the newest version, and its working now, so your 100% correct it is the older version, 10.4 works perfectly.

**Comment by ikuria** [comment]
pls fix that problem, because i cant update my mysql for that :(

**Comment by fahlur** [comment]
I agree with ikuria. I cant update it either stuck with the one my host provides.
mysqlnd 5.0.12-dev

**Comment by tomas-kucera** [comment]
Can I suggest to change the subject of the issue to mysql: search functions do not work ?

This issue does not impact sqlite users.

Thanks!
